.zuul.yaml

```diff
@@ -3,7 +3,67 @@
     name: packit-service/packit-service
     check:
       jobs:
-        - noop
+        - packit-service-pre-commit
+        - packit-service-tests
     gate:
       jobs:
-        - noop
+        - packit-service-pre-commit
+        - packit-service-tests
+
+- import_playbook: ./files/zuul/{{ item }}-secret.yaml
+  with_items:
+  - cert.pem
+  - chain.pem
+  - copr
+  - fullchain.pem
+  - httpd-packit.conf
+  - id_rsa
+  - id_rsa.pub
+  - packit-service.yaml
+  - private-key.pem
+  - privkey.pem
+  - ssh_config
+  - fedora.keytab
+
+- job:
+    name: packit-service-pre-commit
+    parent: base
+    description: Check precommit
+    run: files/zuul-pre-commit.yaml
+    extra-vars:
+      ansible_python_interpreter: /usr/bin/python3
+    nodeset:
+      nodes:
+        - name: test-node
+          label: cloud-fedora-30
+
+- job:
+    name: packit-service-tests
+    parent: base
+    description: Run tests of packit via rpms
+    pre-run:
+      - files/install-openshift.yaml
+    run: files/zuul-tests.yaml
+    extra-vars:
+      with_testing: true
+      ansible_python_interpreter: /usr/bin/python3
+    nodeset:
+      nodes:
+        - name: test-node
+          label: cloud-fedora-30
+    secrets:
```

Comment:
Zuul config elements are not Ansible elements so you cannot use with_items there. Secrets stanza must be a list of Zuul secrets. https://zuul-ci.org/docs/zuul/user/config.html#attr-job.secrets, so something like that:

```
secrets:
  - copr
  - fedora.keytab
  - ...
```
Zuul look at this list and expect to find Zuul secrets with those names.